[torch] remove integer conversion resulted in a change of sign warning

jspark1105 opened the pull request:
Summary: As title

Test Plan: CI

Differential Revision: D21711684

glaringlee reviewed (commented):
@yf225 @jspark1105 
Just curious, is there any benefit we get from this line?
``` int padding_size = padding.size(); ```
Compiler will optimize the .size() anyway to a temp variable here, won't call .size() multiple times.

Update: I see, type conversion warning, nvm then.

glaringlee reviewed (approved):
LGTM.

jspark1105 committed:
[torch] remove integer conversion resulted in a change of sign warning (#38968)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/38968

As title

Test Plan: CI

Reviewed By: glaringlee

Differential Revision: D21711684

fbshipit-source-id: eb8e1c5c2315bad829c859eaed7687ac983eeb1d
